### PR TITLE
Move redirect management into Aurora.

### PR DESCRIPTION
**Pull request by DFurnes**
### Changes
This pull request moves Fastly redirect management out of Nightwing and into Aurora. This means one less admin panel to visit & one less application to keep up to date. Plus, we were able to get some niceties from Aurora for free! 🎨

screen shot 2018-09-20 at 9 14 21 am

References DoSomething/devops#445.

### CODE REVIEW

**Comment by katiecrane** [comment]
Now it seems so obvious! It should always have lived here! 🦇

**Comment by DFurnes** [comment]
Ok two more changes – restricted this resource to admin users (per a conversation with @ngjo) and fixed a little [alignment issue](https://user-images.githubusercontent.com/583202/45826649-dcddc980-bcc2-11e8-9291-89e28c87ac87.png) with confirmation dialog text). Now ready for review! 🙆‍♂️

File: app/Http/Controllers/RedirectsController.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+     */
+    public function destroy(Redirect $redirect)
+    {
+        $successful = $this->fastly->deleteRedirect($redirect->id);
+
+        if (! $successful) {
+            return redirect()->route('redirects.index')->with('flash_message', ['class' => 'messages -error', 'text' => 'Could not delete redirect.']);
+        }
+
+        return redirect()->route('redirects.index')->with('flash_message', ['class' => 'messages', 'text' => 'BAM! Deleted.']);
```
Comment (weerd) [inline app/Http/Controllers/RedirectsController.php]: 💥

File: app/Providers/RouteServiceProvider.php [hunk 2]
```diff
@@ -45,6 +46,16 @@ public function boot()
 
             return $key;
         });
+
+        Route::bind('redirect', function ($key) {
```
Comment (weerd) [inline app/Providers/RouteServiceProvider.php]: Cool!

File: resources/views/auth/unauthorized.blade.php [hunk 3]
```diff
@@ -7,7 +7,8 @@
 		<div class="wrapper">
 			<div class="container__block -narrow">
                 <p><strong>Aurora is only accessible to DoSomething.org staff members.</strong> It doesn't look like your
-                user account has been granted permission to continue. If you think you should, ask in the <code>#api</code>
+                user account has been granted permission to continue. If you
```
Comment (weerd) [inline resources/views/auth/unauthorized.blade.php]: Why not have this as a single line?

File: resources/views/home.blade.php [hunk 4]
```diff
@@ -2,17 +2,16 @@
...
 
     <div class="container -padded">
         <div class="wrapper">
             <div class="container__block -narrow">
                 <p>
-                    This is <strong>Aurora</strong>, our user admin tool. It's the graphical front-end to
-                    <a href="https://github.com/DoSomething/northstar">Northstar</a>, our user & activity API.
-                    Aurora is only available to DoSomething.org staff members.
+                    Welcome to the <strong>DoSomething.org admin interface</strong>. If you're looking to
+                    manage user accounts or domain redirects, you're in the right place.
```
Comment (weerd) [inline resources/views/home.blade.php]: Also here? Single line in code.

File: resources/views/redirects/create.blade.php [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+@include('layout.header', ['header' => 'Redirects', 'subtitle' => 'Create & manage URL redirects.'])
+
+    <div class="container -padded">
+        <div class="wrapper">
+            <div class="container__block -narrow">
+                <h1>Create Redirect</h1>
+
+                @include('layout.errors')
+
+                {!! Form::open(['route' => 'redirects.store', 'method' => 'post']) !!}
```
Comment (weerd) [inline resources/views/redirects/create.blade.php]: We still using `Form` html stuff here. We should add a ticket to remove this all from Aurora at a later date :)

File: resources/views/redirects/edit.blade.php [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+                <h1><code>{{ $redirect->path }}</code></h1>
+
+                @include('layout.errors')
+
+                <form action="{{ route('redirects.update', $redirect->id) }}" method="POST">
+                    <input type="hidden" name="_method" value="PUT">
+                    <input type="hidden" name="_token" value="{{ csrf_token() }}">
+
+                    <div class="form-item -padded">
+                        {!! Form::label('path', 'Incoming Path', ['class' => 'field-label']) !!}
```
Comment (weerd) [inline resources/views/redirects/edit.blade.php]: Mixing html `form` tag with `Form` class stuff here. Maybe just remove the `Form` class stuff and start on the pass to getting rid of the `laravelcollective/html` package :)

**Pull request merged**